bot/cogs/filtering.py

```diff
@@ -468,9 +468,21 @@ async def _has_invites(self, text: str) -> Union[dict, bool]:
                 return True
 
             guild_id = guild.get("id")
-            guild_invite_whitelist = self._get_allowlist_items(True, "guild_invite")
+            guild_invite_whitelist = self._get_allowlist_items("guild_invite", allowed=True)
+            guild_invite_blacklist = self._get_allowlist_items("guild_invite", allowed=False)
 
-            if guild_id not in guild_invite_whitelist:
+            # Is this invite allowed?
+            guild_partnered_or_verified = (
+                'PARTNERED' in guild.get("features")
+                or 'VERIFIED' in guild.get("features")
```

Comment:
Is using `get` necessary here? If it is, keep in mind `x in None` will raise an error, so you'd need a more suitable default value.

Reply:
I guess the answer is, I don't know if it's necessary. I don't know the Discord API well enough to say whether it guarantees that `features` is going to be in the response.

You're absolutely right about the other thing though, refactoring to return empty list.

Reply:
You're right. Better to be safe.